### ISSUE TITLE
Build for M1

### ISSUE THREAD

**Issue by jonesetc** [Ouest-France/terraform-provider-ldap#51]
Would it be possible to add official builds for `darwin_arm64` to the terraform registry? I just switched to a new macbook and when trying to run an init on a project using this provider get:

```
│ Error: Incompatible provider version
│
│ Provider registry.terraform.io/ouest-france/ldap v0.7.6 does not have a package available for your current platform, darwin_arm64.
│
│ Provider releases are separate from Terraform CLI releases, so not all providers are available for all platforms. Other versions of this provider may have different platforms
│ supported.
```

Thank you

**Comment by jonesetc** [comment]
I see that it should be included in the goreleaser action, but it isn't present in the release assets. Is it possible that this is because this is using an older version of goreleaser (`v0.141.0`) before this was resolved? https://github.com/goreleaser/goreleaser/issues/2073

**Comment by jonesetc** [comment]
It wasn't mentioned in that issue, but it does look like this is the case and that it was fixed with release `v0.156.2` https://github.com/goreleaser/goreleaser/releases/tag/v0.156.2

>GOOS=darwin GOARCH=arm64 was ignored before this release, since it was a build for iOS that most people didn't care about and required more than just go to build.
>
>On Go 1.16, this target is now the macOS on Apple Silicon.

I have no experience with goreleaser, so hopefully the project can just move to a newer version of goreleaser specified here: https://github.com/Ouest-France/terraform-provider-ldap/blob/2554b6d45dea24f6dc6baeef4c77c178cd4262cf/.github/workflows/tag.yml#L38

Thank you

**Comment by pablo-ruth** [comment]
Hello @jonesetc,

Sorry for the delay. I upgraded Go and Goreleaser to add support to M1. I released [v0.7.9](https://github.com/Ouest-France/terraform-provider-ldap/releases/tag/v0.7.9) with M1 build.

Tell me if it's ok for you now.

**Comment by jonesetc** [comment]
Can confirm it works on an M1 laptop now, thanks!
